### DISCUSSION TITLE
Disconnecting ExtensionPay from Stripe

### DISCUSSION THREAD

**Discussion by jellohouse** [Glench/ExtPay#177] [Q&A]
Hello,

I had many users asking to pay for the paid versions of my extensions using crypto currencies, so I ended up creating my own server backend for authentication and handling Stripe + crypto payments.

I finally migrated all my extensions to my new system. 

Now I just want to make sure nothing bad would happen when I click Disconnect Stripe in the ExtensionPay dashboard (https://extensionpay.com/home/settings)

I have many users which are paying via Stripe (some from before and some from after the transition).

Again just wanna double check what happens exactly when i disconnect.
It's not gonna cancel all the current subscriptions or delete the products, payment links, customers, subscriptions, etc. in Stripe?

<img width="674" alt="Screen Shot 2023-11-13 at 6 23 35 PM" src="https://github.com/Glench/ExtPay/assets/35238978/75ddf9ac-5c2d-4d17-8ef5-2787254c0ff6">

**Comment by jellohouse** [comment]
Bump @Glench - would like to know your answer on this

**Comment by Glench** [comment]
Nope, it won't delete anything.
